common/js/src/messaging/message-channel-pool.js

```diff
@@ -66,6 +71,16 @@ MessageChannelPool.prototype.getChannels = function(extensionId) {
 };
 
 /**
+ * Returns all the keys in the channels_ multimap, and since a key actually
```

Comment:
You should not reference the private members in documentation or describe how the method is implemented.